frontend/src/components/Header/WideHeader.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faThumbsUp } from '@fortawesome/free-solid-svg-icons';
+import styled from 'react-emotion';
+
+const Content = styled('div')({
+  width: '100%',
+  minHeight: '200px',
+  backgroundColor: '#0a79cd',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+});
```

Comment:
Just a quicktip, if you use the backtick way to write your css elements:
```
const Content = styled('div')`
    ... your css
`;
````
you will be able to use a regular way to write your css:

```
max-width: 20px
```

Reply:
Makes sense. I've been using this one without backticks when there's no need for a variable to do css conditions, as i've seen it throughout the code on phishtray. But for consistency sake, maybe it's better to always use template strings on this. I'll change it.